pythoncapi_compat.h

```diff
@@ -390,7 +390,7 @@ PyThreadState_LeaveTracing(PyThreadState *tstate)
 
 
 // bpo-37194 added PyObject_CallNoArgs() to Python 3.9.0a1
```

Comment:
Would you mind to mention the PyPy version which added support for these functions, here and the same comment below?

Reply:
no problem, I should have known better than to make changes without commenting them ha

pushed it up there